app/src/main/java/com/babylon/wallet/android/presentation/settings/troubleshooting/TroubleshootingSettingsScreen.kt

```diff
@@ -0,0 +1,42 @@
+package com.babylon.wallet.android.presentation.settings.troubleshooting
+
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.WindowInsets
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.statusBars
+import androidx.compose.foundation.lazy.LazyColumn
+import androidx.compose.material3.HorizontalDivider
+import androidx.compose.material3.Scaffold
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import com.babylon.wallet.android.designsystem.theme.RadixTheme
+import com.babylon.wallet.android.presentation.ui.composables.RadixCenteredTopAppBar
+
+@Composable
+fun TroubleshootingSettingsScreen(
+    modifier: Modifier = Modifier,
+    onBackClick: () -> Unit
+) {
+    Scaffold(
+        modifier = modifier.fillMaxSize(),
+        topBar = {
+            RadixCenteredTopAppBar(
+                title = "",
+                onBackClick = onBackClick,
+                windowInsets = WindowInsets.statusBars
+            )
+        },
+        containerColor = RadixTheme.colors.defaultBackground
+    ) { padding ->
+        Column(
+            modifier = Modifier.padding(padding),
+            horizontalAlignment = Alignment.Start
+        ) {
+            HorizontalDivider(color = RadixTheme.colors.gray5)
+            LazyColumn(modifier = Modifier.fillMaxSize()) {
```

Comment:
I guess this still has to be implemented

Reply:
Yes I left that empty for next PR